cli/cmdlineparser.cpp

```diff
@@ -1161,8 +1161,7 @@ void CmdLineParser::printHelp()
               "                         '{file}:{line},{severity},{id},{message}' or\n"
               "                         '{file}({line}):({severity}) {message}' or\n"
               "                         '{callstack} {message}'\n"
-              "                         Pre-defined templates: gcc, vs, edit, cppcheck1\n"
-              "                         The default format is 'gcc'.\n"
+              "                         Pre-defined templates: gcc (default), cppcheck1 (old default), vs, edit, daca2.\n"
```

Comment:
I would prefer that `daca2` is not shown even though it is not "secret". It was not designed to be used by users. If we want to change it for our scripts it would feel good if we can do that without any concern for users.

Reply:
Maybe we can indicate that it is an internal api or something?

Reply:
I'm not sure.. what is the point to show it in the help if it's for internal use.